src/pages/events/[slug].astro

```diff
@@ -118,7 +86,11 @@ const team = [
           </div>
           <h3 class="mt-8 font-serif text-xl font-bold">Wo</h3>
           <a class="mt-3 text-center underline" href="https://maps.app.goo.gl/kak1RX8pJwZGnFFD7">
```

Comment:
This link is still static and not from the cms

Reply:
I've added a TODO comment for now